pkg/controller/useraccount/useraccount_controller_test.go

```diff
@@ -597,8 +608,25 @@ func TestReconcile(t *testing.T) {
 	// Delete useraccount and ensure related resources are also removed
 	t.Run("delete useraccount removes subsequent resources", func(t *testing.T) {
 		// given
+
+		// when the member operator secret exists and has a che admin user configured then che user deletion is enabled
+		restore := test.SetEnvVarsAndRestore(t,
+			test.Env("WATCH_NAMESPACE", "toolchain-member"),
```

Comment:
it's just a detail, but I would use the same namespace as is used in the rest of the test: `test.MemberOperatorNs`
The same applies for the other tests

Reply:
Done